### PR TITLE
PP-4463 Responsible person page: paths, routes and Cypress tests

### PR DESCRIPTION
**Pull request by alexbishop1**
- Add routes and paths so the Stripe responsible person page is generally available
- Add Cypress tests for responsible person page

I know it’s large but it’s mostly fairly short, independent Cypress tests.

### CODE REVIEW

File: test/cypress/integration/stripe-setup/responsible_person_spec.js [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+  const dobMonth = '02'
+  const typedDobMonth = ' 02'
+  const dobYear = '1971'
+  const typedDobYear = '1971 '
+  const friendlyDob = '25 February 1971'
+  const longText = 'This text is 300 ...............................................................................' +
+    '...............................................................................................................' +
+    '............................................................................ characters long'
+
+  const stubStripeAccountGet = function stubStripeAccountGet (stripeAccountId) {
```
Comment (DanailMinchev) [inline test/cypress/integration/stripe-setup/responsible_person_spec.js]: Better naming 👍 Probably we should use similar in the other tests as well

File: test/cypress/integration/stripe-setup/responsible_person_spec.js [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+  const postcode = 'SW15 1LP'
+  const typedPostcode = 'sw151lp '
+  const dobDay = '25'
+  const typedDobDay = '25 '
+  const dobMonth = '02'
+  const typedDobMonth = ' 02'
+  const dobYear = '1971'
+  const typedDobYear = '1971 '
+  const friendlyDob = '25 February 1971'
+  const longText = 'This text is 300 ...............................................................................' +
```
Comment (DanailMinchev) [inline test/cypress/integration/stripe-setup/responsible_person_spec.js]: As far as I calculated it's 299 characters long :D

File: test/cypress/integration/stripe-setup/responsible_person_spec.js [hunk 3]
```diff
@@ -0,0 +1,518 @@
...
+      name: 'getGatewayAccountStripeSetupSuccess',
+      opts: {
+        gateway_account_id: gatewayAccountId,
+        responsible_person: responsiblePersonCompleted
+      }
+    }
+  }
+
+  const stubStripeSetupGetForMultipleCalls = function stubStripeSetupGetForMultipleCalls (...responsiblePersonCompleted) {
+    const data = responsiblePersonCompleted.map(completed => (
```
Comment (DanailMinchev) [inline test/cypress/integration/stripe-setup/responsible_person_spec.js]: Nice 👍 , we should use similar style in the other tests as well

File: test/cypress/integration/stripe-setup/responsible_person_spec.js [hunk 4]
```diff
@@ -0,0 +1,518 @@
...
+      ])
+
+      cy.visit('/responsible-person')
+    })
+
+    it('should display form', () => {
+      cy.get('h1').should('contain', 'Nominate a responsible person')
+
+      cy.get('form[method=post][action="/responsible-person"]').should('exist')
+        .within(() => {
```
Comment (DanailMinchev) [inline test/cypress/integration/stripe-setup/responsible_person_spec.js]: 👍 nice optimisation

File: test/cypress/integration/stripe-setup/responsible_person_spec.js (hunk 1 above)
Comment (alexbishop1) [reply test/cypress/integration/stripe-setup/responsible_person_spec.js]: It dawned on me that “stub” can be used as a verb and then I was off to the races!

File: test/cypress/integration/stripe-setup/responsible_person_spec.js (hunk 2 above)
Comment (alexbishop1) [reply test/cypress/integration/stripe-setup/responsible_person_spec.js]: 😲

File: test/cypress/integration/stripe-setup/responsible_person_spec.js (hunk 3 above)
Comment (alexbishop1) [reply test/cypress/integration/stripe-setup/responsible_person_spec.js]: I’m actually quite proud of this!

File: test/cypress/integration/stripe-setup/responsible_person_spec.js (hunk 4 above)
Comment (alexbishop1) [reply test/cypress/integration/stripe-setup/responsible_person_spec.js]: I know, right? I just stumbled across it in the Cypress docs and was all like:

```css
im so.happy > we [do=not][have~to] have long.selector:prefixes(everywhere)
```

**Pull request merged**